src/rviz/visualization_manager.cpp

```diff
@@ -178,6 +178,10 @@ VisualizationManager::VisualizationManager( RenderPanel* render_panel, WindowMan
                                    "RViz will try to render this many frames per second.",
                                    global_options_, SLOT( updateFps() ), this );
 
+  headlight_property_ = new BoolProperty( "Headlight", true,
+                                          "Default light source attached to camera.",
```

Comment:
The use of camera here can be confused with the Camera display type; I'd suggest "Light source attached to the current 3D view."